sos/report/plugins/scsi.py

```diff
@@ -27,6 +27,7 @@ class Scsi(Plugin, IndependentPlugin):
 
     plugin_name = 'scsi'
     profiles = ('storage', 'hardware')
+    packages = ('sg3_utils')
```

Comment:
This would disable the plugin for all systems without this package, which would not be desirable.